docs/releasing/publishing-a-pre-release.md

```diff
@@ -0,0 +1,135 @@
+
+   - Use `prerelease` to bump from `v5.0.0-beta.0` to `v5.0.0-beta.1`
+   - Use `prerelease` to move from `v5.0.0-internal.X` to `v5.0.0-beta.0`.
+     Note that this resets the final number to `.0`, so prefer setting the version manually if a pre-release of the kind you're running has already gone through.
+
+6. Apply the new pre-release version number by running:
+
+   ```shell
+   npm version <PRE-RELEASE TYPE> --preid <PRE-RELEASE IDENTIFIER> --no-git-tag-version --workspace govuk-frontend
+   ```
+
+   This step will update [`govuk-frontend`'s `package.json`](/packages/govuk-frontend/package.json) and project [`package-lock.json`](/package-lock.json) files.
+
+   Do not commit the changes.
+
+7. Create and check out a new branch (`release-[version]`)
+
+   ```shell
+   git switch -c "release-$(npm run version --silent --workspace govuk-frontend)"
+   ```
+
+8. If you're publishing a beta pre-release, update the [`CHANGELOG.md`](/CHANGELOG.md) by:
+
+   - changing the 'Unreleased' heading to the new version number and release type. For example, '5.0.0-beta.0 (Pre-release)'
+   - adding a new 'Unreleased' heading above the new version number and release type, so users will know where to add PRs to the changelog
+   - saving your changes
+
+9. Run `npm run build-release` to:
+
+   - build GOV.UK Frontend into [the package's `/dist`](/packages/govuk-frontend/dist) and [root `/dist`](/dist) directories
+   - commit the changes
+   - push a branch to GitHub
+
+   You will now be prompted to continue or cancel.
+
+10. Create a pull request.
+    When reviewing the PR, check that the version numbers have been updated and that the compiled assets use this version number.
+
+11. Once a reviewer approves the pull request, merge it to **main**.
+
+## Publish a release to npm
+
+1. Check out the **main** branch and pull the latest changes.
+
+   If there was an interruption between the raising of the PR and its merge,
+   or it's another developer running the publication to npm, rebuild the package with:
+
+   ```shell
+   npm run build:package
+   ```
+
+2. Sign in to npm (`npm login`), using the credentials for the govuk-patterns-and-tools npm user from Bitwarden.
+
+3. Run `npm run publish-release`, which will prompt you to check whether the npm tag looks as expected.
+
+4. Enter `N` to continue to set the npm tag corresponding to the kind of release you're publishing:
+
+   - `internal` for internal pre-releases
+   - `next` for beta pre-releases
+
+5. You will now be prompted to continue or cancel the release. Check the details and enter `y` to continue. If something does not look right, press `N` to cancel the release.
+
+   This step will create a ZIP file containing the release in the root of your govuk-frontend git directory. You will need this file when creating the GitHub release.
+
+   It will also automatically create a tag in GitHub which you can use to create a GitHub release in the following section.
```

Comment:
Pedantic: it's not _super_ clear what "it" is referring to here, since we're talking about a file just before, instead of "this step".